examples/AdminClient/Program.cs

```diff
@@ -54,9 +55,9 @@ static void ListGroups(string brokerList)
         static void PrintMetadata(string brokerList)
         {
             var config = new Dictionary<string, object> { { "bootstrap.servers", brokerList } };
-            using (var producer = new Producer(config))
+            using (var adminClient = new AdminClient(config))
             {
-                var meta = producer.GetMetadata(true, null);
+                var meta = adminClient.QueryMetadata(TimeSpan.FromSeconds(20));
```

Comment:
completely agree. this will require quite a few changes to the return types as well. this api was always marked as "subject to change". might as well do it now.

Reply:
Going to change this back to `GetMetadata` for backward consistency and mark the task as a todo. GetMetadata also returns information about brokers. There is a fair bit of work to change all of this over to a java like api.